src/models/Flashcard.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const flashcardSchema = new mongoose.Schema({
+    topic: {
+        type: String,
+        unique: [true, 'This topic is already taken'],
+        required: [true, 'Cannot be blank'],
+        maxlength: 150,
+    },
+    author: {
```

Comment:
[Question]
Whats the difference between author and createdBy?
If these are effectively the same we can just delete author.

Reply:
I thought we need an author as a field on the card - with the name of the author. The createdBy is an ObjectID (numbers) that won't be seen on the card. I will delete the author field as I don't think that we need it.